test/cctest/testapi.cpp

```diff
@@ -309,6 +309,58 @@ class ShellPlatform : public PlatformRef {
 PersistentRefHolder<VMInstanceRef> g_instance;
 PersistentRefHolder<ContextRef> g_context;
 
+ValueRef* builtinPrint(ExecutionStateRef* state, ValueRef* thisValue, size_t argc, ValueRef** argv, bool isConstructCall)
+{
+    if (argc >= 1) {
+        StringRef* printMsg;
+        if (argv[0]->isSymbol()) {
+            printMsg = argv[0]->asSymbol()->symbolDescriptiveString();
+            puts(printMsg->toStdUTF8String().data());
+            state->context()->printDebugger(printMsg);
+        } else {
+            printMsg = argv[0]->toString(state);
+            puts(printMsg->toStdUTF8String().data());
+            state->context()->printDebugger(printMsg->toString(state));
+        }
+    } else {
+        puts("undefined");
+    }
+    return ValueRef::createUndefined();
+}
+
+ValueRef* builtinTestAssert(ExecutionStateRef* state, ValueRef* thisValue, size_t argc, ValueRef** argv, bool isConstructCall)
+{
+    if (argc >= 2) {
+        EXPECT_TRUE(argv[0]->equalsTo(state, argv[1]));
+    }
+    return ValueRef::createUndefined();
+}
+
+PersistentRefHolder<ContextRef> createEscargotContext(VMInstanceRef* instance)
+{
+    PersistentRefHolder<ContextRef> context = ContextRef::create(instance);
+
+    Evaluator::execute(context, [](ExecutionStateRef* state) -> ValueRef* {
+        ContextRef* context = state->context();
+
+        {
+            FunctionObjectRef::NativeFunctionInfo nativeFunctionInfo(AtomicStringRef::create(context, "print"), builtinPrint, 1, true, false);
+            FunctionObjectRef* buildFunctionObjectRef = FunctionObjectRef::create(state, nativeFunctionInfo);
+            context->globalObject()->defineDataProperty(state, StringRef::createFromASCII("print"), buildFunctionObjectRef, true, true, true);
```

Comment:
It seems that native function `builtinPrint` is not used anywhere.

Reply:
You are right. the print function is just needed for writing test case conveniently.
If you want I can remove the function.